backend/src/rss-feed/rss-feed.controller.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    Controller,
+    Get,
+    Res,
+    Logger
+} from '@nestjs/common';
+import { RssFeed } from './rss-feed.entity';
+import { RssFeedService } from './rss-feed.service';
+
+@Controller('/api/rss-feeds')
+export class RssFeedController {
+
+    constructor(
+        private readonly rssFeedService: RssFeedService,
+        private readonly logger: Logger) {
+        this.logger = new Logger('RssFeedController');
```

Comment:
This is a dependency. The constructor is just initializing it with a name, so that we can identify which logs are from which class otherwise it would be an empty logger without any owner.